onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -720,7 +720,13 @@ TEST(SplitOperatorTest, Split18_InvalidNumOutputs) {
                       3.f, 4.f}});
 
   int64_t num_outputs = 0;
-  RunTest<float>(axis, {}, input, outputs, {kTensorrtExecutionProvider, kQnnExecutionProvider}, true, true, num_outputs, false,
+  const std::unordered_set<std::string> excluded_providers =
+  {
+    kTensorrtExecutionProvider,
+    kQnnExecutionProvider,
+    kDmlExecutionProvider // Error message differs from expected CPU EP error message.
```

Comment:
```suggestion
    kDmlExecutionProvider, // Error message differs from expected CPU EP error message.
```
(minor) Trailing comma helps future diffs to reduce noise of additions/removals :b.